src/app/app.module.ts

```diff
@@ -71,9 +71,10 @@ const matomoBaseUri = 'https://biggive.matomo.cloud';
     {
       provide: RECAPTCHA_LOADER_OPTIONS,
       useValue: {
-        onBeforeLoad(_url: URL) {
+        // see ng-recaptcha/README.md
+        onBeforeLoad(_url: URL): {url: URL, nonce: string} {
           return {
-            baseUrl: 'https://recaptcha.net/recaptcha/api.js', // using this URL instead of default google.com means we avoid google.com cookies.
+            url: new URL("https://www.recaptcha.net/recaptcha/api.js"), // using this URL instead of default google.com means we avoid google.com cookies.
```

Comment:
the library destructures this object assigning url to baseUrl - which means that if url isn't set here it gives a misleading error message about `baseUrl` being undefined.

Reply:
Oops, thanks! I even wondered why I couldn't see a `baseUrl` example on the latest readme, probably should have finished that thought..